### ISSUE TITLE
Possible improvements.

### ISSUE THREAD

**Issue by AndreiCalazans** [PRMarcos/AirCNC#1]
# List of possible improvements which could be made to this codebase.

### Code formatting & Extra spaces
Overall the codebase lacks a pattern of formatting. Take a look at something like [Prettier](https://prettier.io) and ESLINT to help you keep a consistency. Also some JSX code has unnecessary extra space.

### Code

1) Unnecessary function wrapper.

``` 
function handleLogOut() {
    LogOut();
  }
...
<button className="btn" onClick={handleLogOut}>Logout</button>
```

Could be just

``` 
<button className="btn" onClick={LogOut}>Logout</button>
```

2) Uncommon pattern of PascalCase.

Only Components and Models should be Pascal case (Use of Upper case on the first letter). LogOut function should be logOut. 

3) JSX without children

```
return (<Redirect to="/dashboard"></Redirect>) 
```

can be 

```
return <Redirect to="/dashboard" />
```

4) Avoid inlined styles always reference.
`<button style={{ marginBottom: "20px" }} className="btn">Entrar</button>`

should be (name is only example you can use whatever you prefer)

`<button style={styles.margin} className="btn">Entrar</button>`

5) Separate Business logic and Services from UI.

In your Dashboard component you have the following

a) 

```
const DbListening = Db.collection("Booking").where("companyId", "==", currentUser.uid).where("status", "==", "pendding")
            .onSnapshot(function (querySnapshot) {
                var requests = [];
                querySnapshot.forEach(function (doc) {

                    const obj = doc.data();
                    requests.push({ _id: doc.id, ...obj });
                });
                setRequests(requests);

            });
        return () => {
            DbListening();
        }
```

You should write a service function separated from UI component, like `type listenToBookings = (onReceive, onError) => void` 

Same thing for the remaining usage of `Db.collection`  inside this component, by using it directly you are not separating concerns.

## To summarise.

Your code works, it's good, you are doing a good job at building more complex examples like this. Continue at it. It would be nice if you could work on a real project for you to learn about good practices. I recommend you read [this article](https://blog.g2i.co/react-separation-of-concerns-78ec0545bf36) I wrote about separation of concerns and also use things like Prettier and ESLINT to make your code more readable. 

Congratulations.
